### PR TITLE
Created info command

### PR DESCRIPTION
**Pull request by Kyro3400**
Not sure if this is right, might want to make the command ping developers

### CODE REVIEW

File: src/commands/Info.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const BaseCommand = require('../structures/BaseCommand');
+
+class Ping extends BaseCommand {
+  constructor() {
+    super('info');
+  }
+
+  async run(client, message) {
+      message.channel.send('This bot was made by The Mauve-Development Teamn/Xa_Puppet & Jonfirexbox')
+    });
```
Comment (Wilbert-mad) [inline src/commands/Info.js:10]: * Check bracket placement, and exports class

Comment (Kyro3400) [reply src/commands/Info.js:10]: @Wilbert-mad idk what i have to change?

Comment (Wilbert-mad) [reply src/commands/Info.js:10]: @Jonfirexbox Check the class you are exporting and the class name. lastly, check the bracket placement on line 10.
and I'll review this request again please and thank you.

Comment (Kyro3400) [reply src/commands/Info.js:10]: Ok

**Pull request closed without merging**